tensorflow/compiler/xla/service/gpu/tests/gemm_rewrite_test.cc

```diff
@@ -126,12 +126,23 @@ ENTRY AddDotsFunc {
       backend().compiler()->RunHloPasses(
           *get_module(), backend().default_stream_executor(),
           backend().default_stream_executor()->GetAllocator()));
-  StatusOr<bool> filecheck_result = RunFileCheck(optimized_module->ToString(),
-                                                 R"(
+  StatusOr<bool> filecheck_result_cublas =
+      RunFileCheck(optimized_module->ToString(),
+                   R"(
 ; CHECK:    \"selected_algorithm\":\"-1\"
       )");
-  TF_ASSERT_OK(filecheck_result.status());
-  EXPECT_TRUE(filecheck_result.ValueOrDie());
+  TF_ASSERT_OK(filecheck_result_cublas.status());
+
+  // With cublasLt enabled, selected_algorithm is se::blas::kNoAlgorithm
```

Comment:
We probably want quite a few more tests to make sure that cublasLT integration is working?

Reply:
I kept the switch for now since it matches the existing implementation in gemm_thunk.cc.

The GEMM rewriter hasn't changed. The functionality of cuBLASLt can be evaluated by setting the flags and running general GEMM based tests.